misc/loop/scripts/switch.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env sh
+
+set -e
+
+# This script goal is to:
+# 1. Snapshot txs for current running node
+# 2. Start a new node with this tx in genesis file
+# 3. switch traefik node
+
+
+BACKUP_DIR="/backups"
+
+NOW=$(date +'%Y-%m-%d_%s')
+BACKUP_FILE="${BACKUP_DIR}/backup_${NOW}.jsonl"
+BACKUP_LEGACY_FILE=$(echo ${BACKUP_FILE} | sed 's/.jsonl$/-legacy.jsonl/')
+
+HOST_PWD="${HOST_PWD:=$(pwd)}"
+
+TX_ARCHIVE_CMD=${TX_ARCHIVE_CMD:-"tx-archive"}
+
+CONTAINER_NAME="gno-${NOW}"
+
```

Comment:
following command can take a while, add an `echo 'Fetching latest gno container...'`?

Reply:
yeah sorry, forgot to delete this, realised later I was doing something wrong :)